app/_locales/en/messages.json

```diff
@@ -626,6 +626,9 @@
   "buy": {
```

Comment:
Should this be removed if not used anymore?

Reply:
I think we could still reference this in the future, but I'll defer to you. I'm ok with removing it.

Reply:
Just checked the usage, they should be covered in 3 places, we can remove it :)

Reply:
Oh, it is actually required to be removed https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/57869/workflows/a0945dab-6741-491d-8730-0e16cbc93cf6/jobs/1763269

I ran `yarn run verify-locales:fix` here c0518b8bf1